### PR TITLE
Expose loadPromise as method on BaseElement

### PR DESCRIPTION
**Pull request by cramforce**
…instead of importing the module. Almost all extensions use it and it is often their only substatial import.

Significant code size wins across the board.

### CODE REVIEW

File: builtins/amp-img.js [hunk 1]
```diff
@@ -27,7 +26,7 @@ export class AmpImg extends BaseElement {
   constructor(element) {
     super(element);
     /** @private @const {function(!Element, number=): !Promise<!Element>} */
-    this.loadPromise_ = loadPromise;
+    this.loadPromise_ = this.loadPromise;
```
Comment (jridgewell) [inline builtins/amp-img.js]: Is this needed now?

Comment (cramforce) [inline builtins/amp-img.js]: Switched test to use the public member.

**Comment by erwinmombay** [comment]
LGTM

**Comment by cramforce** [comment]
Updated the size after merge.

**Pull request merged**